docs/explanation/cryptography.rst

```diff
@@ -0,0 +1,60 @@
+.. _explanation_cryptographic-technology:
+
+Cryptographic technology in Craft Application
+=============================================
+
+Craft Application uses cryptographic technologies to fetch arbitrary files over the
+internet. It does not directly implement its own cryptography, but it does depend on
+external libraries to do so.
+
+Remote building
+~~~~~~~~~~~~~~~
+
+Craft Application uses `launchpadlib`_ to interact with the `Launchpad`_ API and trigger
+remote builds. Login credentials for Launchpad are stored in an encrypted file in the
```

Comment:
These are stored in plaintext. `cat ~/.local/share/snapcraft/launchpad-credentials`